include/inexor/vulkan-renderer/input/keyboard_mouse_data.hpp

```diff
@@ -26,55 +27,52 @@ class KeyboardMouseInputData {
     KeyboardMouseInputData &operator=(KeyboardMouseInputData &&) = delete;
 
     /// @brief Change the key's state to pressed.
-    /// @param key The key which was pressed.
+    /// @param key the key which was pressed
     void press_key(std::int32_t key);
 
     /// @brief Change the key's state to unpressed.
-    /// @param key The key which was released.
+    /// @param key the key which was released
     void release_key(std::int32_t key);
 
     /// @brief Change the mouse button's state to pressed.
-    /// @param button The mouse button which was pressed.
+    /// @param button the mouse button which was pressed
     void press_mouse_button(std::int32_t button);
 
     /// @brief Change the mouse button's state to unpressed.
-    /// @param button The mouse button which was released.
+    /// @param button the mouse button which was released
     void release_mouse_button(std::int32_t button);
 
     /// @brief Set the current cursor position.
-    /// @param cursor_pos_x The current x-coordinate of the cursor.
-    /// @param cursor_pos_y The current y-coordinate of the cursor.
-    void set_cursor_pos(double cursor_pos_x, double cursor_pos_y);
+    /// @param pos_x the current x-coordinate of the cursor
+    /// @param pos_y the current y-coordinate of the cursor
+    void set_cursor_pos(double pos_x, double pos_y);
 
-    /// @return Return a std::array of size 2 which contains the x-position in index 0 and y-position in index 1.
-    [[nodiscard]] std::array<std::int64_t, 2> get_cursor_pos() const;
+    [[nodiscard]] std::array<std::int64_t, 2> get_cursor_pos();
```

Comment:
I think the `const` can be re-added here now?

Reply:
Done.